apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/app/TrimServiceTest.java

```diff
@@ -137,7 +137,7 @@ void testInitWithExistingOldNotFinishedTrimEntry() {
         doReturn(7199).when(timeService).getEpochTime();
         mockTrimEntries(10000, 11000, 1000);
 
-        trimService.init(11999);
+        trimService.init(11999, 0);
```

Comment:
Please add test to validate new condition `if( lastTrimHeight < shardInitialBlockHeight{`

Reply:
done